app/test/features/chat/custom_input_test.dart

```diff
@@ -106,15 +120,15 @@ void main() {
 
         await tester.enterText(find.byType(TextField), 'testing code');
 
-        await tester.pump();
+        await tester.pump(Durations.medium2);
```

Comment:
Yes, the test throw exceptions of pending timers. I believe it's because of `debounceDelay ` that we also need to mimic that behaviour. I will try to test more and see if I can minimize these delays without failing the tests.